ais/src/main/java/au/gov/amsa/ais/message/AisStaticDataReport.java

```diff
@@ -0,0 +1,11 @@
+package au.gov.amsa.ais.message;
+
+import au.gov.amsa.ais.AisMessage;
+import au.gov.amsa.ais.HasMmsi;
+
+public interface AisStaticDataReport extends AisMessage, HasMmsi {
+
+	int getRepeatIndicator();
+
+	int getPartNumber();
+}
```

Comment:
Is this interface only used by one class? If so remove.

Reply:
It is used by AbstractAisBStaticDataReport and AisStaticDataReportPartA also extends it. Should I remove it?

Reply:
Is it used anywhere (as opposed to just being implemented)? I searched through the changes and couldn't find any use of that interface. On that basis chuck it.

Reply:
Fixed